### DISCUSSION TITLE
Inline edit - checking identical rows before updating

### DISCUSSION THREAD

**Discussion by LukasBreit** [dbeaver/dbeaver#13434] [Ideas]
If you have 2 rows in a table which differ only in one column and the user uses the inline-edit function, he is able to enter the same values for the differing column. So after the changes are saved both rows are completely the same. If you inline-edit one of them afterwards they are also changed both, because they have the same primary key (all columns). Is there a way to check before commiting the inline-edit changes whether there would be completely identical rows after commiting the changes?

**Comment by p91paul** [comment]
If you have any control over this database, I suggest you set a proper primary key and let the database handle duplicate checks.
Implementing this feature might be feasible on small tables, but a real trouble on big ones and IMHO beyond the scope of a db client.
